packages/@okta/vuepress-site/docs/guides/protect-your-api/main/springboot/samplecode.md

```diff
@@ -1,8 +1,7 @@
-See the [Spring Boot Getting Started Guide](https://spring.io/guides/gs/spring-boot/) to build a Spring Boot application to follow the guide with, or download a finished sample app from our [Spring Boot samples repo](https://github.com/okta/samples-java-spring/tree/master/resource-server) or our [WebFlux example repo](https://github.com/okta/okta-spring-boot/tree/master/examples/webflux-resource-server).
-
-You can create a new Spring Boot project with Okta by visiting the [Spring Initializr](https://start.spring.io) and selecting the dependencies **Web**, **Security**, and **Okta**, then clicking **Generate Project** to download a zip file.
-
-You can also use the command line:
+* [Spring Boot Getting Started Guide](https://spring.io/guides/gs/spring-boot/) to build a Spring Boot application to follow the guide with
+* [Spring Boot samples repo](https://github.com/okta/samples-java-spring/tree/master/resource-server) or [WebFlux example repo](https://github.com/okta/okta-spring-boot/tree/master/examples/webflux-resource-server) for a finished sample app that you can download
+* [Spring Initializr](https://start.spring.io) to create a new Spring Boot project with Okta. Select the dependencies **Web**, **Security**, and **Okta**, and then click **Generate Project** to download a zip file.
+* The following command line to create a Spring Boot project:
 
 ```bash
```

Comment:
This seems out of place here? Why a code snippet. Maybe we should build this out into section rather than a bullet point (might be too much effort for this PR; perhaps open a JIRA to address at a later date.)

Reply:
Create a separate Jira ticket to address this out-of-place snippet.